moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/trajectory_monitor.h

```diff
@@ -41,11 +41,11 @@
 #include <moveit/robot_trajectory/robot_trajectory.h>
 #include <boost/thread.hpp>
 #include <memory>
+#include <functional>
 
 namespace planning_scene_monitor
 {
-using TrajectoryStateAddedCallback =
-    boost::function<void(const moveit::core::RobotStateConstPtr&, const rclcpp::Time&)>;
+using TrajectoryStateAddedCallback = std::function<void(const moveit::core::RobotStateConstPtr&, const rclcpp::Time&)>;
```

Comment:
Include-what-you-use would want
```
#include <rclcpp/rclcpp.hpp> // or #include <rclcpp/time.hpp>
```
and whatever declares `moveit::core::RobotStateConstPtr` but I can't seem to find that...

Reply:
I think it is `#include <moveit/robot_state/robot_state.h>`

Reply:
Could be, though I suspect it's that `MOVEIT` macro, which might be there?